v2/test/my_prediction.json

```diff
@@ -0,0 +1 @@
+{"diagnosis_positive": false, "pred_spread": 0.021547846546298433, "model_name": "RandomForestClassifier", "model_training_info": {"n_samples_train": 16, "n_samples_test": 6, "SMOTE": true}, "model_metrics": {"accuracy": 0.5, "balanced_accuracy": 0.5, "log_loss": 17.26965472993207, "precision_score": 0.5, "recall_score": 0.6666666666666666, "f1_score": 0.5714285714285715, "granular_prediction_metrics": {"TP": 2, "TN": 1, "FP": 2, "FN": 1}, "pred_spread": [0.0609527139527139, 0.19118969918969891, 9.79575979564995e-06, 0.08146905871905868, 0.11686824286824277, 0.09138225663225671]}, "feature_importance": {"Diagnosis_tested": 0.0, "Symptoms_cough": 0.07, "Symptoms_body_pain": 0.02, "Symptoms_headache": 0.03, "Symptoms_sore_throat": 0.05, "Symptoms_fever": 0.04, "Symptoms_sinus_pressure": 0.0, "Symptoms_diarrhea": 0.04, "Symptoms_chills": 0.02, "Symptoms_runny_nose": 0.05, "Symptoms_fatigue": 0.12, "Symptoms_reduced_smell_taste": 0.06, "Symptoms_asthma": 0.0, "Symptoms_rash": 0.03, "Symptoms_shortness_breath": 0.01, "Symptoms_sneezing": 0.0, "Symptoms_chest_pain": 0.12, "Symptoms_nausea": 0.02, "Age_11-30": 0.02, "Age_31-50": 0.14, "Age_51-70": 0.03, "Age_nan": 0.0, "Country_Canada": 0.03, "Country_United States of America": 0.04, "Country_nan": 0.0, "Gender_female": 0.04, "Gender_male": 0.02, "Gender_nan": 0.0, "ReportSource_report_diagnosis": 0.0, "ReportSource_nan": 0.0}}
```

Comment:
I think we should think about what will be exposed to users as payload. many of the things like log loss, feature importance is good for our internal analysis, but I feel it will be overwhelming for common users. We can add a link or separate section at the bottom (if advance users want to review the model criteria and performance).

Reply:
Agreed. I figured starting off with "more" would be a good approach. Further, we can display to the user only select criteria. To this I differ to @iwoork + @poattah on the design / front end aspect.